plugin.audio.ilovemusic/resources/senderliste.txt

```diff
@@ -0,0 +1,34 @@
+https://streams.ilovemusic.de/iloveradio1.mp3,I LOVE RADIO,https://raw.githubusercontent.com/adiko01/plugin.audio.ilovemusic/Daten/Senderlogos/i_love_radio.png,
```

Comment:
Please use a resource add-on for this type of content. You can put the logos in that one and use them from this add-on. The images should either be:

* in this add-on
* in another resource add-on within Kodi
* on their original web locations

Having them somewhere else might introduce issues like images being updated outside the add-on. In worst case scenarios to ones that might be offensive to some people.